config/locales/en.yml

```diff
@@ -365,6 +366,11 @@ en:
   settings:
     edit:
       title: Edit settings
+      webauthn_credentials: Security device
+      no_webauthn_credentials: You don't have any security devices
+      webauthn_credential_note: A security device can be can be any device that complies with the FIDO2 standard such as security and biometric keys.
+      webauthn_warning: Once enabled, devices currently can only be used for UI login and are therefore not a replacement for OTP based multi-factor authentication.
```

Comment:
I think we should be more clear that webauthn can be a replacement in the future once API compatibility is completed. Maybe something like:
> Once enabled, devices can only be used for UI login. They are not a replacement for OTP based multi-factor authentication currently. In the future...